changelog.txt

```diff
@@ -1,5 +1,6 @@
 V.Next
 ----------
+- [MINOR] Added support for http intercept of device code request (#1572)
```

Comment:
I'm not sure that this needs a changelog entry, since it's entirely confined to the testing infrastructure.

Reply:
Was running into an error that required an update to the changelog, is there a better way to handle the error?

Reply:
Yes.  That is there to make sure you do a gut check on the changes, and include a changelog entry if needed.  If no entry is required, you can add the "No-Changelog" label, and bypass the check.